code/modules/mob/living/simple_animal/abnormality/waw/snow_whites_apple.dm

```diff
@@ -38,11 +38,14 @@ GLOBAL_LIST_EMPTY(vine_list)
 						)
 	work_damage_amount = 8
 	work_damage_type = BLACK_DAMAGE
-	var/teleport_cooldown
-	var/teleport_cooldown_time = 60 SECONDS
+	initial_language_holder = /datum/language_holder/plant //mostly flavor
 	var/togglemovement
-	var/vine_cooldown
-	var/vine_cooldown_time = 4 SECONDS
+	var/plants_off
+	var/plant_cooldown = 30
+	var/teleport_cooldown
+	var/teleport_cooldown_delay = 60 SECONDS
+	var/unique_cooldown
+	var/unique_cooldown_delay = 4 SECONDS
```

Comment:
"Unique Cooldown" doesn't tell me anything. Why rename it from "Vine Cooldown"?

Reply:
This was a accidentally preperation for another idea people told me. To have quibolith slowing bullets add time to abnormality cooldowns but i should keep that seperated from this.